[38;5;2m[i 0520 13:02:27.509909 32 compiler.py:956] Jittor(1.3.9.14) src: /root/miniconda3/envs/jittordet/lib/python3.8/site-packages/jittor[m
[38;5;2m[i 0520 13:02:27.516194 32 compiler.py:957] g++ at /usr/bin/g++(9.3.0)[m
[38;5;2m[i 0520 13:02:27.516277 32 compiler.py:958] cache_path: /root/.cache/jittor/jt1.3.9/g++9.3.0/py3.8.2/Linux-5.15.0-9xaa/IntelRXeonRSilx76/2997/default[m
[38;5;2m[i 0520 13:02:27.523472 32 __init__.py:412] Found nvcc(11.3.109) at /usr/local/cuda/bin/nvcc.[m
[38;5;2m[i 0520 13:02:27.529720 32 __init__.py:412] Found addr2line(2.34) at /usr/bin/addr2line.[m
[38;5;2m[i 0520 13:02:27.801987 32 compiler.py:1013] cuda key:cu11.3.109_sm_70[m
[38;5;2m[i 0520 13:02:28.434837 32 __init__.py:227] Total mem: 503.29GB, using 16 procs for compiling.[m
[38;5;2m[i 0520 13:02:28.654372 32 jit_compiler.cc:28] Load cc_path: /usr/bin/g++[m
[38;5;2m[i 0520 13:02:28.824531 32 init.cc:63] Found cuda archs: [70,][m
usage: test.py [-h] [--work-dir WORK_DIR] [--disable-cuda] config checkpoint
test.py: error: unrecognized arguments: --eval bbox --samples 10
